tests/helpers/Participant.ts

```diff
@@ -338,6 +339,11 @@ export class Participant {
             && APP.store?.getState()['features/base/participants']?.local?.role === 'moderator');
     }
 
+    async isVisitor() {
+        // TODO: cleaner way to check?
+        return await this.execute(() => APP?.conference?._room?.room?.roomjid?.endsWith('.meet.jitsi'));
```

Comment:
This is dirty as you noted in the comment. In jitsi-meet there is a `iAmVisitor(state)` function - https://github.com/jitsi/jitsi-meet/blob/master/react/features/visitors/functions.ts#L36. Maybe we can use this one?

Since I guess it won't be globally accessible maybe we will need to add it here https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/testing/middleware.ts#L81 and access it trough `window.JitsiMeetJS.app`.

Reply:
Thanks, I used `getState()['features/visitors']?.iAmVisitor` which is the same approach that e.g. `isModerator` uses